config/flags/flags.go

```diff
@@ -86,10 +86,16 @@ var (
 		Usage: "Time interval to regenerate the local transaction journal",
 		Value: evmcore.DefaultTxPoolConfig.Rejournal,
 	}
+	// txpool.pricelimit is deprecated, use txpool.mintip instead
 	TxPoolPriceLimitFlag = cli.Uint64Flag{
 		Name:  "txpool.pricelimit",
+		Usage: "This flag will be DEPRECATED, please use txpool.mintip instead. Minimum gas price limit to enforce for acceptance into the pool",
```

Comment:
The documentation should probably be updated to reflect its actual semantic. If I understand correctly, this flag has actually been controlling the minimum tip all along, right?

Reply:
documentation fixed for both, deprecated and new flag.